INSTALL.md

```diff
@@ -1,68 +1,85 @@
 # Installation
 
-## Clone the repo
+## User install
+
+The quickest way to install GQCP is through conda:
+
+```bash
+conda install -c gqcg -c intel -c conda-forge gqcp
+```
+
+After installation, set the `LIBINT_DATA_PATH` environment variable to the folder that contains these bases. In a default installation (of e.g. version v2.3.1), the data path is given by:
+
+```bash
+export LIBINT_DATA_PATH=${conda_install_dir}/share/libint/2.3.1/basis
+```
+
+## Develop install
+
+###  Clone the repo
 
 Clone the develop branch, which contains the latest developments
 
 ```bash
-    git clone https://github.com/GQCG/GQCP.git --branch develop --single-branch --recurse-submodules
-    cd GQCP
+git clone https://github.com/GQCG/GQCP.git --branch develop --single-branch --recurse-submodules
+cd GQCP
 ```
 
-## Install the GQCG development environment
+### Install the GQCG development environment
 
 Before installing GQCP, please make sure the following dependencies are available on your system:
 
 [![Boost Dependency](https://img.shields.io/badge/Boost-<=1.69-000000.svg)](http://www.boost.org)
 [![Eigen3 Dependency](https://img.shields.io/badge/Eigen-3.3.4+-000000.svg)](http://eigen.tuxfamily.org/index.php?title=Main_Page)
 [![libint2 Dependency](https://img.shields.io/badge/libint-2.3.1+-000000.svg)](https://github.com/evaleev/libint)
 [![libcint Dependency](https://img.shields.io/badge/gqcg_libcint-develop-000000.svg)](https://github.com/GQCG/libcint/tree/develop)
+[![spectra Dependency](https://img.shields.io/badge/gqcg_spectra-master-000000.svg)](https://github.com/GQCG/spectra/tree/master)
```

Comment:
```suggestion
```

Spectra is no longer a dependency for GQCP.

Reply:
I will leave in the Spectra dependency for backwards compatibility. Once we have tagged a release and updated the versioning, I will remove the spectra dependency.

Reply:
As always, you're free to create a release (as long as it's below 1.0.0).